journal.py

```diff
@@ -210,24 +164,25 @@ def editor(request):
 @view_config(route_name='delete')
 def delete(request):
     param = request.matchdict.get('id', -1)
-    request.db.cursor().execute(DELETE_ENTRY, [param])
+    Entry.delete_by_id(param)
     return HTTPFound(request.route_url('home'))
 
 
 @view_config(route_name='details', renderer="templates/details.jinja2")
 def details(request):
-    entry = get_entry(request)
-    entry[0]['text'] = markdown.markdown(entry[0]['text'], extensions=('codehilite', 'fenced_code'))
-    return {'entries': entry}
+    entry = Entry.by_id(request.matchdict.get('id', -1))
+    return {'entry': entry}
 
 
 @view_config(route_name='tweet', renderer='json')
 def tweet_all_about_it(request):
 
-    consumer_key = 'wg0IdOvNPO6SwKNaTz7MWI9J8'
-    consumer_secret = 'OE71lxs2PqOC3hBlIwK85l6npHGsuCmwX0WScAriA4JWFqiVAe'
-    access_token = '3008504700-7ZJGjPtwOZDwEsLHBh4oyoJQdswqmLSyj1QLJI3'
-    access_token_secret = 'UNCrF1nZkurJPpT1ZSACUrGJvwFPxMH4On79Clr611OOa'
+    with open('twitterkeys.txt', 'r') as tk:
+        consumer_key = tk.readline().rstrip()
+        consumer_secret = tk.readline().rstrip()
+        access_token = tk.readline().rstrip()
+        access_token_secret = tk.readline().rstrip()
+    tk.close()
```

Comment:
Oh, and the `with` context manager implies that `tk` will be closed when the with block exits.  I don't think this call is required (and it may throw an error.